src/app/urls.py

```diff
@@ -1,6 +1,10 @@
+
 from django.urls import include, path
 from app import views as app_views
+from . import views
```

Comment:
This will work- I was taught to be more specific (ex. line 3). It helps with tracking down where the code we're referencing is. Can discuss as group if needed.